ui/css/itcss/components/send.scss

```diff
@@ -516,8 +516,8 @@
   }
 
   &__error-hex-data {
-    margin-top: 5px;
-    font-size: 0.7em;
+    @include H8;
```

Comment:
You may get a lint error here. I think there needs to be a line break after mixins :)